main.nf

```diff
@@ -527,7 +527,7 @@ process sample_correlation {
      
      memory { 16.GB * task.attempt }
      time { 2.h * task.attempt }
-     errorStrategy { task.exitStatus == 143 ? 'retry' : 'warning' }
+     errorStrategy { task.exitStatus == 143 ? 'retry' : 'finish' }
```

Comment:
Actually, I think it should be fine. The docs read:
> Initiates an orderly pipeline shutdown when an error condition is raised, waiting the completion of any submitted job.

So I don't think any _new_ processes should be kicked off.